src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
The Lottery class extends ColorSupplier, suggesting an "is-a" relationship. In general, inheritance is used to represent an "is-a" relationship, and it might not be the most appropriate choice in this case. 
The best option use static method from this class.